keras_hub/src/models/mixtral/mixtral_attention.py

```diff
@@ -195,12 +195,9 @@ def _masked_softmax(self, attention_scores, attention_mask=None):
     def _use_fused_attention_op(self):
         if not fused_attention_op_available():
             return False
-        if self.dropout > 0.0:
+        if self._dropout > 0.0:
```

Comment:
Actually, better to leave attrs that map directly to args/configs public, like this https://github.com/keras-team/keras-hub/blob/4bed9ae27db0e0b1b8621c31a320a1c1c08b047c/keras_hub/src/models/gemma/gemma_attention.py#L32-L39